backend/GeoSpatial.sln

```diff
@@ -3,11 +3,13 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29728.190
 MinimumVisualStudioVersion = 15.0.26124.0
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "pims.api", ".\api\pims.api.csproj", "{16BC0468-78F6-4C91-87DA-7403C919E646}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pims.Api", "api\Pims.Api.csproj", "{16BC0468-78F6-4C91-87DA-7403C919E646}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "pims.api.test", ".\test\pims.api.test.csproj", "{19A6829C-20D8-4F66-9889-55C553ACE2B2}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pims.Api.Test", "test\Pims.Api.Test.csproj", "{19A6829C-20D8-4F66-9889-55C553ACE2B2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "pims.dal", ".\dal\pims.dal.csproj", "{6DFFF5E1-1B87-403B-99D0-A9E03D8A8EB3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pims.Dal", "dal\Pims.Dal.csproj", "{6DFFF5E1-1B87-403B-99D0-A9E03D8A8EB3}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Pims.Dal.Entities", "dal\Entities\Pims.Dal.Entities.csproj", "{F6FF4AFE-0B45-469D-93B2-ABD692A9AB2E}"
```

Comment:
Not a standard layout for projects to be within projects.  Normally it would be `/dal/Pims.Dal.csproject` and `/entities/Pims.Dal.Entities.csproject`